tests/WE2E/WE2E_summary.py

```diff
@@ -6,7 +6,13 @@
 
 sys.path.append("../../ush")
 
-from python_utils import load_config_file
+try:
+    from python_utils import load_config_file
+except ModuleNotFoundError:
+    print("\n\nERROR: Could not load python utilities.")
+    print('Note that this script can only be run in the SRW App from the directory:')
+    print("ufs-srweather-app/tests/WE2E\n\n")
+    raise
```

Comment:
Could this not be simplified by using a UW config tool from the API?

Reply:
Most likely, but incorporating uw tools into the WE2E tests is beyond the scope of this PR